web/app/components/new/template-form.hbs

```diff
@@ -0,0 +1,104 @@
+{{! @glint-nocheck: not typesafe yet }}
+{{!-- custom-template-add --}}
+{{#if this.docIsBeingCreated}}
+  <div class="text-center hds-typography-display-400 mt-3">
+    <FlightIcon @name="loading" @size="24" />
+    <div class="mt-8 text-display-200 font-semibold">
+      Creating New Template
+    </div>
+    <div class="text-body-200 text-color-foreground-faint">This usually takes
+      10-20 seconds.</div>
+  </div>
+{{else}}
+  <form
+    class="grid gap-10 grid-cols-[1fr_250px] grid-rows-1"
+    {{on "submit" this.submit}}
+    {{did-insert this.registerForm}}
+  >
+    <div>
+      <div class="space-y-4">
+        <h1
+          class="hds-typography-display-500 hds-font-weight-bold hds-foreground-strong"
+        >Create New Template</h1>
+      </div>
+      {{!-- template id  --}}
+      <div class="pt-10 space-y-7">
+        <Hds::Form::TextInput::Field
+          @type="text"
+          @isRequired={{true}}
+          @value={{this.templateId}}
+          name="templateId"
+          placeholder="Enter new Template Id"
+          {{on "input" this.updateForm}}
+          as |F|
+        >
+          <F.Label>Template ID</F.Label>
```

Comment:
shall it be template name?

Reply:
yeah thanks template name will be better I will change